library/src/scala/tasty/reflect/Kernel.scala

```diff
@@ -550,137 +550,137 @@ trait Kernel {
 
   /** Type tree representing a selection of definition with a given name on a given term prefix */
-  type TypeTree_Select <: TypeTree
+  type TypeSelect <: TypeTree
 
-  def matchTypeTree_Select(tree: Tree)(implicit ctx: Context): Option[TypeTree_Select]
+  def matchTypeSelect(tree: Tree)(implicit ctx: Context): Option[TypeSelect]
 
-  def TypeTree_Select_qualifier(self: TypeTree_Select)(implicit ctx: Context): Term
-  def TypeTree_Select_name(self: TypeTree_Select)(implicit ctx: Context): String
+  def TypeSelect_qualifier(self: TypeSelect)(implicit ctx: Context): Term
+  def TypeSelect_name(self: TypeSelect)(implicit ctx: Context): String
 
-  def TypeTree_Select_apply(qualifier: Term, name: String)(implicit ctx: Context): TypeTree_Select
-  def TypeTree_Select_copy(original: TypeTree_Select)(qualifier: Term, name: String)(implicit ctx: Context): TypeTree_Select
+  def TypeSelect_apply(qualifier: Term, name: String)(implicit ctx: Context): TypeSelect
+  def TypeSelect_copy(original: TypeSelect)(qualifier: Term, name: String)(implicit ctx: Context): TypeSelect
 
   /** Type tree representing a selection of definition with a given name on a given type prefix */
-  type TypeTree_Projection <: TypeTree
+  type Projection <: TypeTree
 
-  def matchTypeTree_Projection(tree: Tree)(implicit ctx: Context): Option[TypeTree_Projection]
+  def matchProjection(tree: Tree)(implicit ctx: Context): Option[Projection]
 
-  def TypeTree_Projection_qualifier(self: TypeTree_Projection)(implicit ctx: Context): TypeTree
-  def TypeTree_Projection_name(self: TypeTree_Projection)(implicit ctx: Context): String
+  def Projection_qualifier(self: Projection)(implicit ctx: Context): TypeTree
+  def Projection_name(self: Projection)(implicit ctx: Context): String
 
-  def TypeTree_Projection_copy(original: TypeTree_Projection)(qualifier: TypeTree, name: String)(implicit ctx: Context): TypeTree_Projection
+  def Projection_copy(original: Projection)(qualifier: TypeTree, name: String)(implicit ctx: Context): Projection
 
   /** Type tree representing a singleton type */
-  type TypeTree_Singleton <: TypeTree
+  type Singleton <: TypeTree
 
-  def matchTypeTree_Singleton(tree: Tree)(implicit ctx: Context): Option[TypeTree_Singleton]
+  def matchSingleton(tree: Tree)(implicit ctx: Context): Option[Singleton]
 
-  def TypeTree_Singleton_ref(self: TypeTree_Singleton)(implicit ctx: Context): Term
+  def Singleton_ref(self: Singleton)(implicit ctx: Context): Term
 
-  def TypeTree_Singleton_apply(ref: Term)(implicit ctx: Context): TypeTree_Singleton
-  def TypeTree_Singleton_copy(original: TypeTree_Singleton)(ref: Term)(implicit ctx: Context): TypeTree_Singleton
+  def Singleton_apply(ref: Term)(implicit ctx: Context): Singleton
+  def Singleton_copy(original: Singleton)(ref: Term)(implicit ctx: Context): Singleton
 
   /** Type tree representing a type refinement */
-  type TypeTree_Refined <: TypeTree
+  type Refined <: TypeTree
```

Comment:
It's called `Refinement` in Type, do we need some consistency here?

Reply:
Will do this renaming after #6193 for simplicity